third_party/objective_c/google_toolbox_for_mac/UnitTesting/GTMGoogleTestRunner.mm

```diff
@@ -0,0 +1,233 @@
+//
+//  GTMGoogleTestRunner.mm
```

Comment:
As noted in the comments below, this class is not compiled as part of the Google Toolbox For Mac project. As this is a standalone file in the project, I chose to just copy over this file instead of pulling in the entire Google Toolbox For Mac project which contains a lot more things that we don't need.

Reply:
Confirm that copying 3rd to our project is ok

Reply:
This is similar to `third_party/objective_c/Cronet/` which is copied from chromium.

Reply:
Right, and nanopb is also the same. But I was just not sure if the differences in licenses would make any difference (sorry I'm totally unfamiliar with that). Doing this might be ok though.

Reply:
Yeah I believe this is ok to do from a licensing POV. Google Toolbox for Mac is [licensed under Apache2](https://github.com/google/google-toolbox-for-mac/blob/master/LICENSE)

Reply:
sg